02-electric/electric-section-three/electric-section-three.html

```diff
@@ -9,37 +9,97 @@
   <title>01 - Electric - Section Three</title>
 </head>
 <body>
-
   <section class="electric-section-three-container">
+
     <section class="electric-section-three-subcontainer">
-      <div class="electric-section-three-img-container">
-        <img
-         class="electric-section-three-img"
-         src="../../00-overall/05-images/placeholder.svg"
-         alt="image section four"
-        />
-      </div>
+
       <h1 class="electric-section-three-title">
```

Comment:
let's wrap this in a container div with the class "electric-section-three-title-container"
and then adjust your css grid template areas to reflect that change

this is to give finer control on the title text later

Reply:
done